lib/travis/build/script/node_js.rb

```diff
@@ -2,10 +2,6 @@ module Travis
   module Build
     class Script
       class NodeJs < Script
-        DEFAULTS = {
```

Comment:
`config[:node_js]` property is the value given in `.travis.yml`. If there is no `node_js` set in `.travis.yml`, it is `nil`.

I _think_ nvm would use whatever is the "default" it thinks if we drop this, but I am not sure. It is safe to have a default value, though, so please add this back.